Frontend/ui-customer/src/Flow.purs

```diff
@@ -6461,4 +6681,72 @@ pickupInstructionsScreenFlow :: FlowBT String Unit
 pickupInstructionsScreenFlow = do
   action <- UI.pickupInstructionsScreen
   case action of 
-    _ -> pickupInstructionsScreenFlow
\ No newline at end of file
+    _ -> pickupInstructionsScreenFlow
+
+customerFeedbackPillData :: RideBookingRes -> String -> Array (Array (Array ST.FeedbackItem))
```

Comment:
@ROHIT-43 These function are already present in HomeScreen/ComponentConfig , Why have you copied and pasted in flow as well?